### DISCUSSION TITLE
Remote include not working in shipping page for SFRA In Development

### DISCUSSION THREAD

**Discussion by sandeepkuma123** [SalesforceCommerceCloud/sfcc-ci#536] [Team Posts]
Hi, 
I have added the content asset in shipping page by using the Page-Include and calling some js event,
this is working only in the local sandbox, not the development and staging and in a console, I am getting like so, what could be the reason for this. 
<wainclude url="removed the site name'/Page-Include?cid=404-page-content
I believe alias will not work for any remote include.

**Comment by pravngaur** [comment]
No, it works fine with aliases also. Just make sure that you have the asset
configured correctly for the given site.

Can you open the asset in a separate tab using the page-show url? If not
then your setup is not correct.

**Comment by cellmanguney** [comment]
Hi
This problem is about response code of 404 page, you cant add page includes with http response 404 or 500, either change the response code at error not found page

**Comment by sandeepkuma123** [comment]
> No, it works fine with aliases also. Just make sure that you have the asset
> configured correctly for the given site.
> 
> Can you open the asset in a separate tab using the page-show url? If not
> then your setup is not correct.
> …

Yes , I did check with the Page-Show and the content is displaying properly, it has the only script.

**Comment by sandeepkuma123** [comment]
> Hi
> This problem is about response code of 404 page, you cant add page includes with http response 404 or 500, either change the response code at error not found page
> …

We are getting the 500 response in the browser console when the event is calling.

**Comment by pravngaur** [comment]
If the reaponse is 500 then its an internal server error, please resolve
the scripting issue first. Once that is fixed your content should appear
fine, as someone mentioned assets don't appear for 404 and 500 response
codes.

**Comment by sandeepkuma123** [comment]
Thanks, @pravngaur, It got resolved.
